app/actions/SanityCheckAction.js

```diff
@@ -85,11 +84,10 @@ export const cancelDBCleanup = (sanityStatus) => {
 };
 
 export const flagCleanupComplete = (isStartMainApp) => {
-  logger.log(`flagCleanupComplete: isStartMainApp = ${isStartMainApp}`);
-  logger.log(`Sanity check duration (possibly with alerts): ${DateUtils.duration(start, new Date())}`);
+  console.log(`flagCleanupComplete: isStartMainApp = ${isStartMainApp}`);
```

Comment:
Do we need to log to the console, or can we use logger? (I don't know if its already initialize by now)